src/status_im/ui/screens/views.cljs

```diff
@@ -23,6 +28,9 @@
         (let [component (case current-view
                           :login login
                           :profile profile
+                          :new-chat new-chat
```

Comment:
Can we already render these screens (chat, new-chat, new-public-chat)?

Reply:
@vkjr not yet, but I'm looking what is failing for the "chat" screen